### PR TITLE
DO-1951 Generate action for SPDX > XLS

### PR DESCRIPTION
**Pull request by duhow**

### CODE REVIEW

File: oss-license-generate-xls/action.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        # Add headers
+        echo '"OSS COMPONENT NAME","VERSION","PACKAGE LICENSE","EXTRA DATA"' > "${TMPCSV}"
+
+        # Create csv for each xls tab. Insert v in version to avoid being undestood as a date in XLS
+        jq '.packages | .[] | {(.name): {version: ("v" + .versionInfo), license: .licenseConcluded}}' "${REPORT}" | jq -s .[] | jq -s  add > "${TMPJSON}"
+        jq -rs '.[0] * .[1] | to_entries [] | [.key,.value.version,.value.license,.value.extraData] |@csv' "${TMPJSON}" "${CUSTOM}" >> "${TMPCSV}"
+
+        ssconvert -T Gnumeric_Excel:xlsx2 --merge-to=${OUTPUT} ${TMPCSV}
+
+        rm -f ${ŤMPJSON} ${TMPCSV}
```
Comment (arponpes) [inline oss-license-generate-xls/action.yaml:61]: That's a weird T haha
```
>>> ord('Ť')
356
>>> ord('T')
84
```

**Review by Sturgelose — APPROVED** [review]
LGTM!

Fast question and Low hanging Fruit: Could we create a GitHub Action that groups the four github Actions we have created together?

So, this way, usage is just configuring a single action instead of 4 different ones.
If someone ever needs more fine-grain control, they can use the low-level ones :)

File: oss-license-generate-xls/action.yaml (hunk 1 above)
Comment (duhow) [reply oss-license-generate-xls/action.yaml]: Great catch, thanks!

**Comment by duhow** [comment]
@Sturgelose unfortunately no.
This action will be used for `evp-device-agent` repository, the other repositories do not have the manual SBOM file and is not expected for the time being.

**Review by arponpes — APPROVED** [review]

**Review by midokura-agustin — APPROVED** [review]
LGTM

**Pull request merged**